tests/test_for_support/test_for_sbo.py

```diff
@@ -70,10 +81,18 @@ def test_find_components_without_sbo_terms(model, num, components):
     ("specific_sbo_term", 0, "metabolites"),
     ("specific_sbo_term", 1, "reactions"),
     ("specific_sbo_term", 1, "genes")
+    ("multiple_sbo_terms", 0, "metabolites"),
+    ("multiple_sbo_terms", 1, "reactions"),
+    ("multiple_sbo_terms", 1, "genes")
 ], indirect=["model"])
 def test_find_components_without_specific_sbo_term(model, num, components):
```

Comment:
I think it may be cleaner to take the `term` variable up into the parameters and only have a single assert statement in the test.